pylabrobot/plate_reading/biotek_backend.py

```diff
@@ -1164,13 +1164,25 @@ def image_size(magnification: float) -> Tuple[float, float]:
       # "wide fov" is an option in gen5.exe, but in reality it takes the same pictures. So we just
       # simply take the wide fov option.
       # um to mm (plr unit)
-      if magnification == 4:
-        return (3474 / 1000, 3474 / 1000)
-      if magnification == 20:
-        return (694 / 1000, 694 / 1000)
-      if magnification == 40:
-        return (347 / 1000, 347 / 1000)
-      raise ValueError(f"Don't know image size for magnification {magnification}")
+
+      if self.version.startswith("1"):
+        if magnification == 2.5: #actually 2.74x
+          return (3160 / 1000, 2370 / 1000) # pixel size = 2.45 um
+        if magnification == 4:
+           return (2164.8 / 1000, 1619.5 / 1000) # pixel size  = 1.68 um
+        if magnification == 20:
+            return (433 / 1000, 325 / 1000)  # pixel size ≈ 0.336 µm 
+        if magnification == 40:
+            return (216.5 / 1000, 161.9 /1000)  # pixel size ≈ 0.168 µm
+        raise ValueError(f"Don't know image size for magnification {magnification}")
+      else:
+        if magnification == 4:
+          return (3474 / 1000, 3474 / 1000)
+        if magnification == 20:
+          return (694 / 1000, 694 / 1000)
+        if magnification == 40:
+          return (347 / 1000, 347 / 1000)
```

Comment:
do you know this is because of cytation 1? it seems to me tis would be determined by the sensor (since it's possible to share sensors between cytation1 and 5 & why would they record different sizes on a different machine?)

Reply:
also `else` after return/raise is not necessary :)